libdebug/debugger/debugger.py

```diff
@@ -82,137 +82,155 @@ def wait(self: Debugger) -> None:
         """Waits for the process to stop."""
         self._internal_debugger.wait()
 
+    def maps(self: Debugger) -> list[MemoryMap]:
```

Comment:
Maps is missing a "search" funcion, like d.maps("libc") fuzzy-matching maps with similar backing file names

Reply:
open an issue with your propose, please

Reply:
I thought we have already decided to implement this feature, but okay

Reply:
The issue is also important for tracking purposes, as it can be integrated into the github project. Otherwise, I might forget it again. :)
Moreover, we should decide on the API to expose to do that.